src/elements/content-sidebar/activity-feed/comment-form/CommentForm.js

```diff
@@ -129,7 +129,7 @@ class CommentForm extends React.Component<Props, State> {
 
         return (
             <Media className={inputContainerClassNames}>
-                {!isEditing && (
+                {!isEditing && !!user && !!getAvatarUrl && (
```

Comment:
Should we pull this logic out into a separate variable for readability?

Reply:
Tried doing that but it gives some linting errors